metricbeat/module/elasticsearch/pending_tasks/_meta/fields.yml

```diff
@@ -0,0 +1,25 @@
+- name: cluster.pending_tasks
+  type: group
+  description: >
+    `cluster.pending_tasks` contains pending tasks description.
+  fields:
+    - name: time_in_queue
```

Comment:
What is the unit of this one? In general I would prefer that we only have 1 field, best the one with more accuracy and set the type as postfix.

Reply:
What are the units of the two fields?

Reply:
Thanks for the link. Had a look at the document and I think we should drop `time_in_queue` is this mainly for human consumption and instead use `time_in_queue_millis` and name it `time_in_queue.ms`. Like this we should have both in one field.